server/.env

```diff
@@ -1 +1,5 @@
 TEAM_MEMBERS="Aecio, Bonnie, Ritesh"
+
+DATABASE=salescrm
+DATABASE_USER=postgres
+DATABASE_PASSWORD=myPassword
```

Comment:
We will want to gitingore server after this PR now

Reply:
I dont understand this part. Do we gitignore the .env file or the whole server folder after we merge the above codes?

Reply:
I think he means the .env file, since everyone would have PostgreSQL configured in a different way.

Reply:
Yes the .env file - sorry for the confusing message above. In addition this will contain some private information when you work on deployment